src/pheval_exomiser/prepare/write_application_properties.py

```diff
@@ -0,0 +1,172 @@
+        ):
+            self.application_properties.write(
+                "exomiser.hg38.cadd-snv-path="
+                "${exomiser.data-directory}/cadd/${cadd.version}/hg38/"
+                "whole_genome_SNVs.tsv.gz\n"
+            )
+
+    def write_exomiser_hg38_cadd_indel_path(self) -> None:
+        """Write the hg38 cadd indel path to application.properties file."""
+        if (
+            self.configurations.application_properties.cadd_version is not None
+            and self.configurations.application_properties.hg38_data_version is not None
+        ):
+            self.application_properties.write(
+                "exomiser.hg38.cadd-in-del-path="
+                "${exomiser.data-directory}/cadd/${cadd.version}/hg38/"
+                "InDels.tsv.gz\n"
+            )
+
+    def write_exomiser_hg38_remm_path(self) -> None:
+        """Write the hg38 remm path to application.properties file."""
+        if (
+            self.configurations.application_properties.remm_version is not None
+            and self.configurations.application_properties.hg38_data_version is not None
+        ):
+            self.application_properties.write(
+                "exomiser.hg38.remm-path="
+                "${exomiser.data-directory}/remm/ReMM.v${remm.version}.hg38.tsv.gz\n"
+            )
+
+    def write_exomiser_hg38_local_frequency_path(self) -> None:
+        """Write the hg38 local frequency path to application.properties file."""
+        if self.configurations.application_properties.hg38_local_frequency_path is not None:
+            self.application_properties.write(
+                f"exomiser.hg38.local-frequency-path="
+                f"${{exomiser.data-directory}}/local/"
+                f"{self.configurations.application_properties.hg38_local_frequency_path}\n"
+            )
+
+    def write_exomiser_phenotype_data_version(self) -> None:
+        """Write the phenotype data version to application.properties file."""
+        self.application_properties.write(
+            f"exomiser.phenotype.data-version={self.configurations.application_properties.phenotype_data_version}\n"
+        )
+
+    def write_hg19_white_list_path(self) -> None:
+        """Write the hg19 whitelist path to application.properties file."""
+        if self.configurations.application_properties.hg19_data_version is not None:
+            self.application_properties.write(
+                "exomiser.hg19.variant-white-list-path=${exomiser.hg19.data-version}_hg19_clinvar_whitelist.tsv.gz\n"
+            )
+
+    def write_hg38_white_list_path(self) -> None:
+        """Write the hg38 whitelist path to application.properties file."""
+        if self.configurations.application_properties.hg38_data_version is not None:
+            self.application_properties.write(
+                "exomiser.hg38.variant-white-list-path=${exomiser.hg38.data-version}_hg38_clinvar_whitelist.tsv.gz\n"
+            )
+
+    def write_cache_spec(self):
```

Comment:
also need the `spring.cache.type` as this will enable caffeine, assuming caffeine is enabled. I other words, if `spring.cache.type=caffeine`, then `spring.cache.caffeine.spec=maximumSize` needs to be set. 

```properties
### caching ###
# If you're running exomiser in batch mode there might be some performance benefit if you enable caching. The 'simple'
# option will continue to store data in memory *without* limit - this means for really long-running batch jobs and/or
# whole genomes you may run out of memory.
# If this is likely choose the caffeine option and uncomment spring.cache.caffeine.spec and adjust the cache size
# to your requirements
#none/simple/caffeine
#spring.cache.type=none
#spring.cache.caffeine.spec=maximumSize=60000
```